changelog.md

```diff
@@ -1,5 +1,18 @@
 # CHANGELOG.md
 
+## 0.2.0 (2019-08-23)
+
+Breaking changes:
+
+  - `unmock.on()` returns `this` instead of `states` object.
+  - Deleted `unmock.states`: state changes should be done via `unmock.services["service"].state`
```

Comment:
`unmock.services.service.state`? (or do you think that's implied?)

Reply:
I just thought using the `[]` accessor made it a bit more clear that the `service` part is not part of the API but essentially a variable.